server.js

```diff
@@ -27,6 +27,34 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+// Returning all the books (objects) from the array 
+app.get('/books', (req, res) => {
+  const { page } = req.query
+  const startIndex = 20 * +page
```

Comment:
It would be nice with a default here. If I call the API without a `page` query param, I don't get any results, but if I pass `page=0` then I get the first page. You could use `||` (or) to set a default if `+page` is `NaN`:

```
const startIndex = 20 * +page || 0
```